### ISSUE TITLE
Support DAC

### ISSUE THREAD

**Issue by EvanCarroll** [dbcli/mssql-cli#93]
I don't see the `-A` working which is the option on the Microsoft platform for sqlcmd.exe, nor do I see the `admin:localhost` syntax which is the method of connecting to DAC on sqlcmd.exe on Linux. Is there going to be support for that method?

**Comment by pensivebrian** [comment]
Thanks for filing this issue. We'll look into adding support for DAC connections, and will update this issue when we have a timeline for the fix.

**Comment by EvanCarroll** [comment]
Cool, just for reference when you try to connect with `admin:localhost` you can pass auth but then you get a cryptic error,

```
$ mssql-cli -S admin:localhost
Username (press enter for sa): 
Password: 
Version: 0.8.0
Mail: sqlcli@microsoft.com
Home: http://github.com/dbcli/mssql-cli
master>SELECT 1;
Query failed: BeginExecuteReader requires an open and available Connection. The connection's current state is closed.
Time: 0.051s
```

**Comment by pensivebrian** [comment]
This should be fixed in 0.9.0 and later. Running a `pip install --upgrade mssql-cli` command should upgrade to a version with the fix.
